### PR TITLE
Add requirements.txt

### PR DESCRIPTION
**Pull request by jae2**
For those of us who prefer to use ansible in a virtualenv this makes things nice and easy for us as we can just do pip install -r requirements.txt.

Steps to test:

```
virtualenv venv 
source venv/bin/activate
pip install -r requirements.txt
ansible-playbook -i your_inventory playbooks/path/to/playbook
```

### CODE REVIEW

**Comment by detiber** [comment]
@abutcher ptal

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+enum34==1.1.2
+idna==2.0
+ipaddress==1.0.16
+paramiko==1.16.0
+pyOpenSSL==0.15.1
+pyasn1==0.1.9
+pycparser==2.14
+pycrypto==2.6.1
+six==1.10.0
+wsgiref==0.1.2
```
Comment (abutcher) [inline requirements.txt]: Should we leave versions out of the requirements except for the ansible version? Most of these are dependencies of ansible. Can we limit the requirements to ansible and pyopenssl?

**Comment by DanyC97** [comment]
@jae2 are you still working on this? seems is a trivial change following @abutcher review . Happy to take it if you don't have time to do so

**Comment by sdodson** [comment]
@DanyC97 looks like no response from OP so i've closed this.

**Pull request closed without merging**